docs/stylesheets/base/utilities.css

```diff
@@ -6,3 +12,11 @@
   gap: 20px;
   padding: 10rem;
 }
+
+.showcase-avatar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  margin-top: 2rem;

```

Comment:
Try to use padding more times than margin. It is good practice as padding is always inside the content box in box-model

Reply:
Thanks, Tahir
Will change it